src/smart-components/group/role/group-roles.js

```diff
@@ -88,6 +88,8 @@ const GroupRoles = ({
   const [showRemoveModal, setShowRemoveModal] = useState(false);
   const [confirmDelete, setConfirmDelete] = useState(() => null);
   const [deleteInfo, setDeleteInfo] = useState({});
+  const { userAccessAdministrator, orgAdmin } = useContext(PermissionsContext);
+  const hasPermissions = useRef(orgAdmin || userAccessAdministrator);
```

Comment:
I like the usage of ref in here (no need to trigger yet another re-render when `orgAdmin` or `userAccessAdministrator` changes) I am a bit concerned about the change of these values. There is no effect looking at these calues and changing the current value of `hasPermissions`. I know these values can't change while sitting on the page, but what about initial render?

Reply:
@karelhala good point, it could cause problems. I added `useEffect`, which monitors the value as you suggest. 
Thank you :slightly_smiling_face: